networks.json

```
@@ -141,6 +169,10 @@
     "42161": {
       "address": "0x812308712a6d1367f437e1c1e4af85c854e1e9f6",
       "transactionHash": "0x3f15799dbf9c2901bcef125df4cd10b3942144d17d3b876a6f47b1653ebed900"
+    },
+    "232": {
+      "address": "0x812308712a6D1367f437E1c1e4af85C854E1e9f6",
```

Consistent capitalization:

```suggestion
      "address": "0x812308712a6d1367f437e1c1e4af85c854e1e9f6",
```